scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1208,6 +1213,81 @@ static bool keep_running(struct cpu_ctx *cctx, struct task_struct *p)
 	return false;
 }
 
+bool consume_delayed_dsq(u64 dsq_id, u64 now_sec) {
+	struct task_struct *p;
+	struct task_ctx *tctx;
+	u64 runnable_at_sec;
+	
+	if (!dsq_id || !now_sec)
+		return false;
+	
+	bpf_for_each(scx_dsq, p, dsq_id, 0) {
+		if ((tctx = lookup_task_ctx(p))) {
+			if (tctx->runnable_at) {
+				runnable_at_sec = tctx->runnable_at / CONFIG_HZ;
+					scx_bpf_error("%llu %llu", now_sec, runnable_at_sec);
```

Comment:
debugging and this looks odd?

this line of log prints bpf_jiffies64/CONFIG_HZ and tctx->runnable_at/CONFIG_HZ
```
Error: EXIT: scx_bpf_error (14335776 65064116979)
```

Reply:
I think this is relevant to that (making sense of that still):
```
❯ sudo cat /proc/timer_list  | grep jif 
  .idle_jiffies   : 4301162074
  .last_jiffies   : 4301162075
jiffies: 4301162075
  .idle_jiffies   : 4301161701
  .last_jiffies   : 4301161701
```

Reply:
no idea what task_ctx.runnable_at is but the one to use is the one on the task struct

Reply:
Yeah, `task_ctx->runnable_at` is set by layered from `bpf_ktime_get_ns()`. `p->scx.runnable_at` is the one in jiffies. Alternatively, as `task_ctx->runnable_at` exists already, you can just use `bpf_ktime_get_ns()` as `now` too.